spec/features/07_user_logins_facebook_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "user visits their show page" do
+  let!(:user_1) do
+    User.create(
+      email: 'jscho13@gmail.com',
+      name: 'Joseph Cho',
+      gender: 'Male',
+      password: 'Applied91'
+    )
+  end
+
+  let!(:t_url) do
+    'https://ariherzog.com/blog/wp-content/uploads/2010/10/longurl.jpg'
+  end
+
+  before(:each) do
+    OmniAuth.config.mock_auth[:facebook] = nil
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+      provider: 'facebook',
+      uid: '123545',
+      info: {
+        email: 'jscho13@gmail.com',
+        name: 'Joseph Cho',
+        image: t_url
+      },
+      extra: { raw_info: { gender: 'Male' } }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.